spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -100,15 +102,47 @@ def get_message(self):
 
         return message
 
+    @staticmethod
+    def get_tags(message):
+        tags = {}
+        failed_monitors = message.get("failed_monitors", [])
+        failed_monitors = [y.split("/") for y in failed_monitors]
+        for key, group in groupby(failed_monitors, key=lambda x: x[0]):
+            for mon in group:
+                try:
+                    k = mon[1].lower().replace(" ", "_")
```

Comment:
**issue**: When failed method name has special characters (like `á`, `*`, etc), the tag is not applied correctly and Sentry adds an error in the notification. 

![invalid_chars](https://user-images.githubusercontent.com/382887/105064568-b7605680-5a5b-11eb-92ae-c8010c28627f.png)

**suggestion**: It would be great if we handle these specials characters to avoid this problem.

Reply:
Ouch. I wonder how should we escape them (dropping them looks bad).

Reply:
**suggestion**: Maybe we may consider using a library like [python-slugify](https://github.com/un33k/python-slugify) to generate proper slug for this string, removing/replacing unwanted characters. This library is already a spidermon dependency (for [validation stats](https://github.com/scrapinghub/spidermon/blob/master/spidermon/contrib/scrapy/stats.py#L2)), so you can use it with no need to include an extra requirement to the project.